src/builtin.d

```diff
@@ -48,35 +49,35 @@ extern (C++) Expression eval_sin(Loc loc, FuncDeclaration fd, Expressions* argum
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, sinl(arg0.toReal()), arg0.type);
+    return new RealExp(loc, TargetReal.sin(arg0.toReal()), arg0.type);
```

Comment:
> That type doesn't seem to offer sin/cos/tan/sqrt, so that's going to be an issue

By the way, these functions are implemented by the MPFR library, which requires gcc to convert their "``real_t``" into an ``mpfr_ptr`` then back again after calling the function.  :-)